src/Defaults/DefaultPlug.vala

```diff
@@ -121,7 +120,7 @@ public class Defaults.Plug : Gtk.Box {
             }));
 
             // TRANSLATORS: This is description for for screen reader. %s can be "Web Browser" or "Music Player" and so on.
-            app_chooser.get_accessible ().accessible_name = _("Default %s").printf (label);
+            // app_chooser.get_accessible ().accessible_name = _("Default %s").printf (label);
```

Comment:
Not sure how to set this in GTK 4?

Reply:
I tried `app_chooser.update_property (Gtk.AccessibleProperty.LABEL, _("Default %s").printf (label), -1);` and I tried `            ((Gtk.Label) setting_label.get_first_child ()).mnemonic_widget = app_chooser;` with no result